### PR TITLE
Separate label for /run/systemd/notify

### PR DESCRIPTION
**Pull request by dsugar100**
label systemd_runtime_notify_t
Allow daemon domains to write by default

### CODE REVIEW

**Comment by dsugar100** [comment]
This is intended as an alternative approach for #692 

It will need some review and testing.  There are other domains that use init_write_runtime_socket that may need to be updated to systemd_write_notify_socket like podman, kuberneties, docker, logrotate.

File: policy/modules/system/systemd.if [hunk 1]
```diff
@@ -274,6 +275,25 @@ interface(`systemd_user_unix_stream_activated_socket',`
...
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`systemd_write_notify_socket',`
+	gen_require(`
+		type systemd_runtime_notify_t;
+	')
+
+	allow $1 systemd_runtime_notify_t:sock_file write;
```
Comment (pebenito) [inline policy/modules/system/systemd.if:294]: It needs:
* the dir accesses to get to the sock_file
* a unix socket connectto/sendto init_t.

**Review by pebenito — CHANGES_REQUESTED** [review]

File: policy/modules/system/systemd.if (hunk 1 above)
Comment (dsugar100) [reply policy/modules/system/systemd.if:299]: Ok, I did both of these.  The dir access makes sense.  I'm not sure I understand the unix socket change and I think I did what you are asking.  Of course, if I misunderstood, let me know and I will alter it.

Comment (dsugar100) [reply policy/modules/system/systemd.if:298]: Ok, just the sendto.  Hopefully this is now good.

**Review by pebenito — APPROVED** [review]

**Pull request merged**